vbox: add import-vbox-vm

Bogdanp opened the pull request:
Replaces #3

mflatt commented:
This seems generally fine and should be merged (finally), but is there a reason to convert paths to strings? Leaving them as paths has the advantage of working with unusual paths that don't round-trip through strings.

The documentation needs `history` note, and the check of #5 should be added, but I'm happy to do those.

Bogdanp commented:
Thanks! I must've gotten confused looking at the contracts of `system*` and `system` and thought the former only accepted strings. I've fixed that, added a history note and bumped the version. Feel free to take over if you want to make other changes.
